### PR TITLE
Shreeyash garde patch 1

### PR DESCRIPTION
**Pull request by ShreeyashGarde**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4053"></a>

### CODE REVIEW

File: data/ShreeyashGarde.json [hunk 1]
```diff
@@ -18,6 +18,8 @@
     },
     {
       "name": "Connect with me on Github",
+
+      "url": "https://github.com/ShreeyashGarde",
       "url": "https://github.com/settings/profile",
```
Comment (Panquesito7) [inline data/ShreeyashGarde.json:23]: Duplicate link.
```suggestion
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by ShreeyashGarde** [comment]
Changes done

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by SaraJaoude — APPROVED** [review]
looks good

**Comment by ShreeyashGarde** [comment]
Thanks

**Pull request merged**